sergey.kiselev/task-2-1/cmd/service/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"sergey.kiselev/task-2-1/internal/temperature"
+)
+
+var errArgument = errors.New("invalid argument")
+
+func processEmployee(manager *temperature.TemperatureManager) error {
+	var (
+		operator    string
+		temperature int
+	)
+
+	if _, err := fmt.Scan(&operator, &temperature); err != nil {
+		return fmt.Errorf("invalid input: %w", err)
+	}
+
+	if err := manager.Update(operator, temperature); err != nil {
+		return fmt.Errorf("%w", err)
+	}
+
+	fmt.Println(manager.GetComfortTemp())
+
+	return nil
+}
+
+func processDepartment() error {
+	var countEmployees int
+
+	if _, err := fmt.Scan(&countEmployees); err != nil {
+		return fmt.Errorf("invalid input: %w", err)
+	}
+
+	if countEmployees < 1 {
+		return fmt.Errorf("%w: employee count must be positive", errArgument)
+	}
+
+	manager := temperature.New()
+
+	for range countEmployees {
+		if err := processEmployee(manager); err != nil {
+			return fmt.Errorf("%w", err)
```

Comment:
обернуть ошибку - значит добавить контекст... но так как это метод из того же пакета, и если тут не нужен котекст - можно просто 


```suggestion
			return err
```

Reply:
<img width="1395" height="177" alt="image" src="https://github.com/user-attachments/assets/2d7518ac-9227-4566-8b51-3f461d7628bf" />
либо я дурачок, либо линтер не оценил ваше идею